.evergreen/scripts/install-uninstall-check.sh

```diff
@@ -13,14 +13,21 @@ CMAKE=$(find_cmake_latest)
 . $DIR/check-symlink.sh
 SRCROOT=$(pwd)
 
+# The major version of the project. Appears in certain install filenames.
+_full_version=$(cat "$DIR/../../VERSION_CURRENT")
+version="${_full_version%-*}"  # 1.2.3-dev → 1.2.3
+major="${version%%.*}"         # 1.2.3     → 1
+echo "major version: $version"
+echo " full version: $major"
```

Comment:
```suggestion
echo "major version: $major"
echo " full version: $version"
```

Same suggestion applies to other occurrences of these `echo`s.